pyfenix/__main__.py

```diff
@@ -1,16 +1,28 @@
 """Fenix runner"""
 
-import queue
+import asyncio
+from queue import Queue
 import threading
 
-from tests.doubles import ClientAPIStub, GUIEventHandlerStub
+from tests.doubles import GUIEventHandlerStub
 
 from .client_event_loop import ClientEventLoop
+from .ws_api import WebsocketsAPI
 
-event_queue: queue.Queue = queue.Queue()
-api = ClientAPIStub(event_queue)
-gui = GUIEventHandlerStub(event_queue)
+async def main() -> None:
+    """Run Fenix"""
+    event_queue: Queue = Queue()
+    api = WebsocketsAPI(event_queue)
+    await api.connect(("localhost", 60221))
+    await api.send("yay")
+    asyncio.create_task(api.listen())
 
-handler = ClientEventLoop(event_queue, api, gui)
-handler_thread = threading.Thread(target=handler.run)
-handler_thread.start()
+    gui = GUIEventHandlerStub(event_queue)
+
+    handler = ClientEventLoop(event_queue, api, gui)
+    handler_thread = threading.Thread(target=handler.run)
+    handler_thread.start()
+
+    await api.close()
+
+asyncio.get_event_loop().run_until_complete(main())
```

Comment:
```suggestion
if __name__ == "__main__":
    asyncio.get_event_loop().run_until_complete(main())
```

Reply:
This is redundant in a `__main__.py` file because it will always be run as "__main__".